tests/test_build_epub.py

```diff
@@ -320,13 +320,12 @@ def test_epub_anchor_id(app):
     app.build()
 
     html = (app.outdir / 'index.xhtml').read_text()
-    assert ('<p id="std-setting-staticfiles_finders">'
-            '<span id="std-setting-STATICFILES_FINDERS"></span>'
+    assert ('<p id="std-setting-STATICFILES_FINDERS">'
             'blah blah blah</p>' in html)
-    assert ('<span id="std-setting-staticfiles_section"></span>'
+    assert ('<span id="std-setting-STATICFILES_SECTION"></span>'
             '<span id="std-setting-STATICFILES_SECTION"></span>'
```

Comment:
It seems these node_ids are duplicated. I need to investigate where second ID came from.